front/src/common/SpeedLimitByTagSelector/useStoreDataForSpeedLimitByTagSelector.ts

```diff
@@ -4,13 +4,14 @@ import { compact, concat, uniq } from 'lodash';
 import { useTranslation } from 'react-i18next';
 import { useSelector } from 'react-redux';
 
+import COMPOSITION_CODES from 'assets/compositionCode/compositionCodes';
```

Comment:
This import seems to still reference the old file?

Reply:
Done